lib/connect/ws.js

```diff
@@ -1,6 +1,9 @@
 'use strict'
 
-var websocket = require('websocket-stream')
+import * as dbg from 'debug'
```

Comment:
This should not have landed.

Reply:
I forced a commit to the branch, hence why it landed. This PR was not into Master though. Just a carry-over PR into another branch...

Reply:
don't use import :).